common/src/provisioning/Authentication/IAuthorizationHeaderProvider.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.Azure.Devices.Common.Service.Auth
+namespace Microsoft.Azure.Devices.Provisioning.Service
 {
     internal interface IAuthorizationHeaderProvider
     {
```

Comment:
Same as above - this got moved, namespace changed but nothing else but service client is using it. What's the reason it resides in common?

Reply:
Same, not being used; has been removed